### DISCUSSION TITLE
[Help] How to unstick a sticky element

### DISCUSSION THREAD

**Discussion by donaggio** [fomantic/Fomantic-UI#1433] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Is there a way to un-stick a sticky element previously initialized with something like:

`$('.ui.sticky').sticky({ ... });`

There are some situations (when reflowing content on viewport resizing, for example) when previously sticky elements need not to be sticky anymore.

There doesn't seem to be any behaviour to do just that (or I haven't been able to find it in the docs).

**Comment by dutrieux** [comment, marked as answer]
I think `$('.ui.sticky').sticky("destroy");` is enough

**Comment by donaggio** [comment]
I've tried that, but it doesn't seem to do anything. Setting 'debug' to true and calling the 'destroy' behavior doesn't print any message on the console (should output 'Destroying previous instance' according to the sticky module sources).

**Comment by donaggio** [comment]
Nevermind, it works, I was calling the 'destroy' behavior at the wrong time, before even initializing the element with .sticky({ ... }) !
